modules/ossm-threescale-webassembly-module-services-object.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assembly:
+//
+// service_mesh/v2x/ossm-threescale-webassembly-module.adoc
+
+[id="ossm-threescale-webassembly-module-services-object_{context}"]
+= The 3scale WebAssembly module `services` object
+
+The `services` top-level object specifies which service identifiers are handled by this particular instance of the `module`.
+
+Since accounts can have multiple services, you must specify which ones are handled. The rest of the configuration revolves around how to configure services.
+
+The `services` field is required. It is an array that should contain at least one service to be useful.
+
+----
+services:
+  - id: "2555417834789"
+    token: service_token
+    authorities:
+      - "*.app"
+      - 0.0.0.0
+      - "0.0.0.0:8443"
+    credentials: <object_see_below_table>
+    mapping_rules: <object_see_below_table>
+----
+
+Each element in the `services` array represents a 3scale service.
+
+.The following are `services` object fields:
+|===
+|Name |Description |Required
+
+|`ID`
+|An identifier for this 3scale service, currently not referenced elsewhere.
+|Yes
+
+|`token`
```

Comment:
@rahulanand16nov Does the extension support autoloading token and other stuff from 3scale?

Reply:
@unleashed mentioned he's working on fetching the details from the 3scale dashboard, but AFAIK, PR is not up yet (or at least I can't find it in the merged ones).

Reply:
Perhaps it was a work in progress before Alex went on leave.

Do we need a note or some other update to let the customer know specifics here or is an engineering PR needed to remedy this?

Reply:
@rahulanand16nov We synced with Alex and we decided it would be best to postpone the feature to the next release.

@dfennessy For documentation it will mean a few additions and explanations, which I will add comments for. Regarding this token I would add something like this:
This Token can be found in proxy configuration (ideally a link to 3scale doc) under `backend_authentication_value` key or by using this command `curl "<tenant_url>/admin/api/services/<service_id>/proxy/configs/production/latest.json?access_token=<access_token>" | jq '.proxy_config.content.backend_authentication_value'`

Reply:
@pehala, I got the same message from Alex as well, but I'll try to push the code, and If I am able to finish it, we can re-amend this part in the docs.

Reply:
Thanks for the information @pehala 

It's not immediately clear to me where I link to in the 3scale docs. I looked, but I don't see any reference to `backend_authentication_value` under proxy configuration.

Reply:
Ideally to doc containing reading using proxy configuration (if such exists). There is no doc mentioning  `backend_authentication_value` so that would have to be handled definitely here.

Reply:
Thanks @pehala The only 3scale doc I see that mention proxy configuration are here: https://access.redhat.com/documentation/en-us/red_hat_3scale_api_management/2.10/html-single/operating_3scale/index#listing-proxy-configurations

Are these the correct docs? 

@rahulanand16nov Can documentation about `backend_authentication_value` be added to the upstream docs regarding the token?